source/geod24/LocalRest.d

```diff
@@ -1843,3 +1860,44 @@ unittest
     node2.ctrl.shutdown();
     thread_joinAll();
 }
+
+///
+unittest
+{
+    static import core.thread;
+    import core.time;
+
+    static interface API
+    {
+        public void startTimer ();
+        public ulong getCounter ();
+    }
+
+    static class Node : API
+    {
+        public override void startTimer ()
+        {
+            setTimer(100.msecs, &callback, /* periodic: */ true);
+        }
+
+        public void callback () @safe
+        {
+            this.counter++;
+        }
+
+        public override ulong getCounter ()
+        {
+            scope (exit) this.counter = 0;
+            return this.counter;
+        }
+
+        private ulong counter;
+    }
+
+    auto node = RemoteAPI!API.spawn!Node();
+    assert(node.getCounter() == 0);
+    node.startTimer();
+    core.thread.Thread.sleep(1.seconds);
+    assert(node.getCounter() >= 8);
```

Comment:
```suggestion
    assert(node.getCounter() >= 2);
```

2 should be enough (it means the timer repeated). 8 is too close to the possible upper limit of 10 (10 calls every 100 msecs), and if the machine is too slow then it might have called the timer fewer times than 8.

Reply:
Apply the suggestion.